### ISSUE TITLE
Swimming FP

### ISSUE THREAD

**Issue by Kyrobi** [CaptainObvious0/NoCheatPlus#28]
### Full output of /ncp version
---- Version information ----
#### Server ####
git-Paper-588 ~MC: 1.13.2~
  detected: 1.13.2
#### NoCheatPlus ####
Plugin: 3.16.1-SNAPSHOT-b1153-CustomFork
MCAccess: 1.4.5-1.12.2|? / CB-Reflect
Features:
  blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13
  checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
  defaults: pvpKnockBackVelocity
  packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance
Hooks: AllViolations~NCP~ 1.0 | mcMMO~cncp~ 2.3
#### Related Plugins ####
CompatNoCheatPlus v6.6.6-SNAPSHOT-sMD5NET-b91 | ProtocolLib v4.4.0 | ProtocolSupport v4.29-dev

### Describe the issue
When trying to moving move on water (not swimming), you get flagged for survival fly.

### How to reproduce the issue
Go and move on water

### Extra links/Videos (Including debug logs)
Debug Link: https://imgur.com/a/psjbKzX
Video: https://youtu.be/0Wg4dde6Fbw

### Any possible config options changed or plugins that may cause interference?
No config change.

**Comment by CaptainObvious0** [comment]
Duplicate of #23 

Issue has already been fixed.
